### PR TITLE
convert back into bytes before putting into csv file

### PR DESCRIPTION
**Pull request by erwinmombay**
Closes https://github.com/ampproject/amphtml/issues/6275

### CODE REVIEW

File: build-system/tasks/csvify-size/index.js [hunk 1]
```diff
@@ -174,6 +176,22 @@ function mergeTables(dateTimes, tables) {
 }
 
 /**
+ * @param {string} prettyBytes
+ * @return {number}
+ */
+function reversePrettyBytes(prettyBytes) {
+  var triple = prettyBytes.match(/(\d+(?:\.\d+)?)\ (B|kB|MB|GB|TB|PB|EB|ZB|YB)/);
```
Comment (jridgewell) [inline build-system/tasks/csvify-size/index.js]: Why is `kB` lowercase but everything else is uppercase?

LOL, are we gonna push TB files?!

File: build-system/tasks/csvify-size/index.js [hunk 2]
```diff
@@ -174,6 +176,22 @@ function mergeTables(dateTimes, tables) {
...
+  var value = triple[1];
+  var unit = triple[2];
+
+  if (!(value && unit)) {
+    return 0;
+  }
+  var exponent = prettyBytesUnits.indexOf(unit);
+  return (Number(value) * Math.pow(1000, exponent)).toFixed(3);
```
Comment (jridgewell) [inline build-system/tasks/csvify-size/index.js]: 1000, or 1024?

Comment (erwinmombay) [inline build-system/tasks/csvify-size/index.js]: i reversing what pretty bytes does https://github.com/sindresorhus/pretty-bytes/blob/master/index.js#L20 so should be 1000 i think

File: build-system/tasks/csvify-size/index.js (hunk 1 above)
Comment (erwinmombay) [inline build-system/tasks/csvify-size/index.js]: looks like it's using decimal table instead of binary

**Pull request merged**